packages/secp256k1/secp256k1.0.3.2/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "Davide Gessa <gessadavide@gmail.com>"
+authors: [
+  "Davide Gessa <gessadavide@gmail.com>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "Yoichi Hirai <i@yoichihirai.com>"
+]
+homepage: "https://github.com/dakk/secp256k1-ml"
+bug-reports: "https://github.com/dakk/secp256k1-ml/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/dakk/secp256k1-ml"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build & >= "1.0+beta9"}
+  "base" {build & >= "v0.9.2"}
+  "stdio" {build & >= "v0.9.0"}
+  "configurator" {build & >= "v0.9.1"}
+  "hex" {test & >= "1.1.1"}
+  "ounit" {test & >= "2.2.2"}
+  "conf-secp256k1"
+  "base-bigarray"
+]
+
```

Comment:
please add a minimum OCaml version (and a maximum version if applicable.

For example, if you're using jbuilder your minimum version should at the very least be 4.02.3

Reply:
@rgringer I'm not sure it's a good idea to add minimum versions from your dependencies. If the package is using jbuilder, and jbuilder correctly indicates its availability information, why duplicate it in this package? New jbuilder versions may support more or less ocaml versions, which would make the bound here obsolete.

Reply:
I agree with @gasche; the dependency solver will automatically stop secp256k1 in all deps are not satisfied, so if jbuilder is not installable, secp256k1 inherits this behavior